### ISSUE TITLE
mnist_saved_model.py: NameError: name 'xrange' is not defined

### ISSUE THREAD

**Issue by liuqun** [tensorflow/serving#988]
https://github.com/tensorflow/serving/blob/524f1bf87d39be3b0ba212b30aa82f899a09adb1/tensorflow_serving/example/mnist_saved_model.py#L75

Since Python-3.5 no longer supports `xrange()`, we would get the following error while runnng [the serving_basic example](https://tensorflow.google.cn/serving/serving_basic) inside a tensorflow docker container:

```
# python tensorflow_serving/example/mnist_saved_model.py /tmp/mnist_model/
Training model...
Extracting /tmp/train-images-idx3-ubyte.gz
Extracting /tmp/train-labels-idx1-ubyte.gz
Extracting /tmp/t10k-images-idx3-ubyte.gz
Extracting /tmp/t10k-labels-idx1-ubyte.gz
2018-07-13 03:06:42.027122: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
Traceback (most recent call last):
  File "tensorflow_serving/example/mnist_saved_model.py", line 147, in <module>
    tf.app.run()
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/platform/app.py", line 125, in run
    _sys.exit(main(argv))
  File "tensorflow_serving/example/mnist_saved_model.py", line 75, in main
    tf.constant([str(i) for i in xrange(10)]))
NameError: name 'xrange' is not defined
```

```
# python --version
Python 3.5.2

# python -c "import tensorflow ; print(tensorflow.__version__)"
1.9.0
```

```
$ docker image ls
REPOSITORY              TAG                 IMAGE ID            CREATED             SIZE
tensorflow/tensorflow   latest-py3          66da89656f5f        2 days ago          1.24GB

$ docker container ls
CONTAINER ID        IMAGE                              COMMAND             CREATED             STATUS              PORTS                NAMES
5484e3dd2fcc        tensorflow/tensorflow:latest-py3   "/bin/bash"         About an hour ago   Up About an hour    6006/tcp, 8888/tcp   tfcontainer

$ uname -a
Linux debian 3.16.0-6-amd64 #1 SMP Debian 3.16.56-1+deb8u1 (2018-05-08) x86_64 GNU/Linux
```

**Comment by gautamvasudevan** [comment]
Unfortunately, the examples are not py3 compatible yet.

**Comment by liuqun** [comment]
@gautamvasudevan  PR #991 is a patch for py3 now.

**Comment by gautamvasudevan** [comment]
Fixed by #991

**Comment by gautamvasudevan** [comment]
Thanks for the patch!
